src/components/Capture/CaptureForm/CaptureForm.tsx

```diff
@@ -22,6 +22,8 @@ import {
   CaptureError
 } from './CaptureFormStyled';
 
+const PHONE_NUMBER_REGEX = /(^[+]?([0-9][-|" "]?){4,16})$/;
```

Comment:
i think this could be placed in some `constants` file

Reply:
Moved this and other regular expressions to new `regexConstants.ts` file